cmd/root.go

```diff
@@ -162,6 +162,9 @@ func createFn(cmd *cobra.Command, args []string) error {
 	if err := t.Push(cmd.Context()); err != nil {
 		return err
 	}
+	if err := t.CheckNodeStatus(cmd.Context()); err != nil {
```

Comment:
so I am not sure we "always" want to wait here i would feel this likely should be a flag